robyn/__init__.py

```diff
@@ -165,6 +166,40 @@ def terminating_signal_handler(_sig, _frame):
                 observer.stop()
                 observer.join()
 
+    def add_view(self, endpoint: str, view: Callable, const: bool = False):
+        """
+        [This is base handler for the view decorators]
+
+        :param endpoint [str]: [endpoint for the route added]
+        :param handler [function]: [represents the function passed as a parent handler for single route with different route types]
```

Comment:
I'm not very familiar with docstrings but I think there is no need for the `[]`? We don't have them in other parts of the code

Reply:
Yep true. :D